app/styles/about.sass

```diff
@@ -1,36 +1,323 @@
+@import "app/styles/bootstrap/variables"
+@import "app/styles/mixins"
+@import "app/styles/style-flat"
+
 #about-view
```

Comment:
Due to 3c3db79ac1bf5117f4d048da07b4fe690ca066dd, and the fact that this page has infinite banners, you'll need to add overflow: hidden to the view itself, otherwise the banners expand the page rightward.